StortSpelProjekt/Game/src/main.cpp

```diff
@@ -171,7 +171,7 @@ Scene* GameScene(SceneManager* sm)
     melc = entity->AddComponent<component::MeleeComponent>();
```

Comment:
on row 172, could you please add this:     melc->SetDamage(10);
this somehow got lost in merges, thank you!